### DISCUSSION TITLE
No sound when recording a video in QuickLook

### DISCUSSION THREAD

**Discussion by justlovejazz** [google/model-viewer#3856] [Q&A]
### Description

In ar mode on ios there is no sound when recording a video, although everything is ok on android.
Any suggestions for this issue.

### Browser Affected

- [ ] Chrome, version: xx.x.xxxx.xx
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [x] Safari

### OS

- [ ] Android
- [x] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### AR

- [ ] WebXR
- [ ] SceneViewer
- [x] QuickLook

**Comment by elalish** [comment]
We have no control over Apple products 😄 Please file a bug on WebKit (that's Safari, but we've had pretty good luck getting QuickLook issues addressed there). @grorg for visibility.
